CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
-# Changelog
+# CHANGELOG
```

Comment:
Nit: Can we remove trailing periods from changelog entries?

Reply:
removed!